packs/data/other-effects.db/effect-2-circumstance-penalty-to-attack-rolls.json

```diff
@@ -0,0 +1,48 @@
+{
+    "_id": "5gX3PZztMecmWxX9",
+    "img": "systems/pf2e/icons/effects/critical-effect.webp",
+    "name": "Effect: -2 circumstance penalty to attack rolls",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #10]{Critical Fumble Deck #10}, @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #15]{Critical Fumble Deck #15}, @UUID[Compendium.pf2e.criticaldeck.Critical Hit Deck #29]{Critical Hit Deck #29}</strong></p>\n<p>-2 circumstance penalty to attack rolls</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Critical Effect",
+                "selector": "attack",
+                "slug": "critical-effect-penalty-to-attack-2",
+                "type": "circumstance",
+                "value": -2
+            }
+        ],
+        "source": {
+            "value": ""
```

Comment:
```suggestion
            "value": "Pathfinder Critical Fumble Deck"
```

Reply:
I set these to Pathfinder Critical Decks as several effects are shared across the Critical Hit and Critical Fumble decks. Is this ok or does the source *have* to point to a single specific "source/product"?